libc/src/string/memmem.cpp

```diff
@@ -15,10 +15,10 @@ namespace LIBC_NAMESPACE {
 LLVM_LIBC_FUNCTION(void *, memmem,
                    (const void *haystack, size_t haystack_len,
                     const void *needle, size_t needle_len)) {
-  constexpr auto comp = [](unsigned char l, unsigned char r) -> int {
+  constexpr auto COMP = [](unsigned char l, unsigned char r) -> int {
```

Comment:
Is this change wanted ?

Reply:
Yes, although it could be done as a separate PR.